stubs/Symfony/Component/Security/Acl/Model/AclProviderInterface.stub

```diff
@@ -0,0 +1,16 @@
+ <?php
```

Comment:
There is a space in the beginning of the line 😄 

```suggestion
<?php
```

Reply:
With PHPStan's dev-master, the experience is a bit nicer:

```
------ --------------------------------------------------------------------------------------------
  Line   stubs/Symfony/Component/Security/Acl/Model/AclProviderInterface.stub
 ------ --------------------------------------------------------------------------------------------
  3      Namespace declaration statement has to be the very first statement in the script on line 3
  3      Namespace declaration statement has to be the very first statement in the script on line 3
  3      Namespace declaration statement has to be the very first statement in the script on line 3
  3      Namespace declaration statement has to be the very first statement in the script on line 3
 ------ --------------------------------------------------------------------------------------------
```

Reply:
Whoah, nice catch